Marlin/Configuration_adv.h

```diff
@@ -656,6 +656,22 @@
   // Swap the CW/CCW indicators in the graphics overlay
   //#define OVERLAY_GFX_REVERSE
 
+  #if ENABLED(U8GLIB_ST7920)
```

Comment:
Would it make more sense to put the check for a U8GLIB_ST7920-based display into the LIGHTWEIGHT_UI code, then put a "LIGHTWEIGHT_UI requires a U8GLIB_ST7920 based graphical display." into SanityCheck.h ?

How will users know they have a U8GLIB_ST7920-based display without digging deeper beyond both Configuration*.h and the board pin*.h files ?

(I mention this because I ran into this confusion by setting LIGHTWEIGHT_UI on what I thought was a U8GLIB_ST7920 based LCD and not seeing any change, but turns out it was an U8GLIB_SSD1306 LCD).

Reply:
The configurator uses these `#if` wrappers to decide whether to present the option. But we should also add a sanity check.